rest/taskana-rest-spring/src/main/java/pro/taskana/workbasket/rest/WorkbasketController.java

```diff
@@ -136,6 +138,27 @@
     return result;
   }
 
+  /**
+   * This endpoint deletes an existing Workbasket.
+   *
+   * <p>Returned HTTP Status codes:
+   *
+   * <ul>
+   *   <li><b>204 NO_CONTENT</b> - Workbasket has been deleted successfully
```

Comment:
Didn't we want something like this for the other endpoints as well?

Reply:
Yes we do. We agreed on doing this in a seperate ticket.